apps/tng-dictionary/src/components/Home/Home.tsx

```diff
@@ -1,120 +1,126 @@
-import './Home.module.css';
-import * as React from 'react';
-import { Link } from "react-router-dom";
-import { Backdrop, Button, Divider } from '@mui/material';
-import SearchIcon from '@mui/icons-material/Search';
 import AndroidIcon from '@mui/icons-material/Android';
 import AppleIcon from '@mui/icons-material/Apple';
 import InfoOutlinedIcon from '@mui/icons-material/InfoOutlined';
-import BackdropUnstyled from '@mui/base/BackdropUnstyled';
+import SearchIcon from '@mui/icons-material/Search';
+import { Backdrop, Button, Divider } from '@mui/material';
 import { motion } from 'framer-motion';
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import './Home.module.css';
 
 /* eslint-disable-next-line */
-export interface HomeProps { }
+export interface HomeProps {}
```

Comment:
Can you remove this if there are no props?

Reply:
Fixed